python/common/org/python/types/Slice.java

```diff
@@ -35,8 +35,13 @@ public Slice(org.python.Object start, org.python.Object stop, org.python.Object
         }
 
         if (step instanceof org.python.types.Int) {
-            this.__dict__.put("step", step);
             this.step = (org.python.types.Int) step;
+            if (this.step.value == 0 ){
```

Comment:
There's some odd spacing around this closing brace - the space should be *after* the `)`, not before.